src/ussd_clinic.js

```diff
@@ -84,14 +97,112 @@ go.app = function() {
             }
         };
 
+        self.compile_registrant_info = function() {
+            var registrant_info = self.im.user.answers.registrant;
+            registrant_info.details.lang_code = self.im.user.answers.lang_code;
+            registrant_info.details.consent =
```

Comment:
Yes, but I'll open an issue for it since it's accross different apps that have pull requests open at the moment